pkg/run.go

```diff
@@ -46,10 +46,10 @@ func RunFromConfig(conf *config.Config) (types.AssessmentMap, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), config.Conf.Timeout)
 	defer cancel()
 
-	return run(ctx)
+	return Run(ctx)
 }
 
-func run(ctx context.Context) (ret types.AssessmentMap, err error) {
+func Run(ctx context.Context) (ret types.AssessmentMap, err error) {
```

Comment:
If you are going to use this directly then you'll have to manually know to set config.Conf like in RunFromConfig, I think it might be better to introduce: `RunWithContext(ctx context.Context, conf *config.Conf)`

Reply:
@Tehsmash thank you, that's a good point. I've introduced the `RunWithContext` function.